tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/amqp/connect/AMQPReplicaTest.java

```diff
@@ -640,25 +640,32 @@ private void replicaTest(boolean largeMessage,
          server_2.startBrokerConnection(brokerConnectionName);
       }
 
+      snfreplica = server_2.locateQueue(replica.getMirrorSNF());
```

Comment:
Why do you need to repopulate the variable, shouldnt it still be the same?

Reply:
the broker restarted in some of the tests.. so it's better to repopulate it.

Reply:
But not this test? Its populated 11 lines earlier, just before the connection is stopped and started for "making sure we wouldn't duplicate the mirror". I couple maybe see adding an assertion here that doing a new locate returns the exact same queue object, i.e that the connection stop/start process didnt disturb the queue, but I dont really see reason to just reset the variable.